### PR TITLE
Changes to landing page

### PR DESCRIPTION
**Pull request by vermontbail**
When going through the wiki, I realized that the landing page of the site didn't have much detail, only redirection to other sections of the site. I went over the front-facing material from both the Github and the discord to enhance the home page into something that could be directly shared to someone interested, even before they engaged with the Github. It also combined some changes I made when merging the old repo, if those are bad in any way you can discard them. I am only focused on the changes within index.md.

### CODE REVIEW

File: docs/Developers/Building-Locally.md [hunk 1]
```diff
@@ -14,34 +14,28 @@ So first things first - the initial "No assumptions" setup to have Unciv run fro
     -   Clone your fork with git - the location will be https://github.com/YourUsername/Unciv.git, visible from the green "Clone or download" button at https://github.com/YourUsername/Unciv
 -   Load the project in Android Studio, Gradle will attempt the initial sync. If this is your first time with Android Studio, this may require you to accept the Android Build-tools licenses, which works differently on every device, so search for your OS-specific solution.
     -   A new install may not be able to do the initial sync - this comes in the form of `Unable to find method ''void org.apache.commons.compress.archivers.zip.ZipFile.<init>(java.nio.channels.SeekableByteChannel)''` errors when you try to sync. If you have this problem go into File > Settings > Appearance & Behavior > System Settings > Android SDK
-        - Click "SDK Platforms"
-        - Click "Android 12L (Sv2)"
-        ![image](/docs/assets/Android_SDK_Platforms.png)
-        - Click "SDK Tools"
-        - Select "Show Package Details" in the bottom right
-        - Choose version 32.0.0 under "Android SDK Build-Tools"
-        ![image](/docs/assets/Android_SDK_Tools.png)
```
Comment (yairm210) [inline docs/Developers/Building-Locally.md:23]: Looks like your PR removes changes that were made to add these images?

**Comment by yairm210** [comment]
The changes to the index.md look good, I'll accept this PR if you remove touching the 'building locally' page :)

**Comment by vermontbail** [comment]
How would I go about reverting those changes in Github? Like reverting the first two commits so it just contains index.md.

**Comment by yairm210** [comment]
Pulling this 'main' into yours may work

**Pull request closed without merging**